src/components/Snackbar/Snackbar.types.ts

```diff
@@ -0,0 +1,39 @@
+import { IconName } from '../Icon';
+import React from 'react';
+import { ButtonProps } from '../Button';
+
+export enum SnackbarType {
+    neutral = 'neutral',
+    positive = 'positive',
+    warning = 'warning',
+    disruptive = 'disruptive',
+}
+
+export type CloseButtonProps = Omit<ButtonProps, 'onClick' | 'icon'>;
+
+export type SnackbarPosition =
+    | 'top'
+    | 'top-left'
+    | 'top-right'
+    | 'bottom'
+    | 'bottom-left'
+    | 'bottom-right';
+
+export interface SnackbarProps {
```

Comment:
As discussed over chat, adding a scoped hook and exposing a handler here to support older apps could be a good approach in addition to the exported hook.